web/js/projection/projection.ui.js

```diff
@@ -1,15 +1,8 @@
-/**
- * @module wv.proj
- */
-var wv = wv || {};
-wv.proj = wv.proj || {};
+import $ from 'jquery';
+import each from 'lodash/each';
+import wvui from '../ui/ui';
 
-/**
- * Undocumented.
- *
- * @class wv.proj.ui
- */
-wv.proj.ui = wv.proj.ui || function (models, config) {
+export function projectionUi(models, config) {
```

Comment:
What do you think about keeping the name `ui`, importing this into the `proj` parent, and attaching it to the object that exports?